datumaro/components/progress_reporting.py

```diff
@@ -146,9 +146,15 @@ def start(self, total: int, *, desc: Optional[str] = None):
 
     def report_status(self, progress: int):
         status = str(self._desc) if self._desc else ""
-        status += f"{progress}/{self._total} ({progress/self._total*100:.2f}%)"
+        status += (
+            f" {progress:0{len(str(self._total))}d}/{self._total}"
```

Comment:
Do we need `progress:0` <- `"0"`?

Reply:
Would you elaborate it more? I don't get what you mean.

Reply:
Sorry, I was mistaken that it was outside the bracket, e.g. `f"progress:0{len(str(self._total))}d}/{self._total}"`